pages/studentAssessmentExam/studentAssessmentExam.sql

```diff
@@ -2,11 +2,10 @@
 SELECT
     ai.*
 FROM
-    assessment_instances AS ai,
-    (SELECT *
-     FROM group_users AS gi 
-     WHERE $user_id = gi.user_id) AS gid
+    assessment_instances AS ai
+    LEFT JOIN groups AS g ON (g.deleted_at IS NULL)
+    LEFT JOIN group_users AS gu ON (gu.group_id = ai.group_id AND gu.group_id = g.id)
```

Comment:
I think the following is equivalent and it feels a bit more natural to me:

```suggestion
    assessment_instances AS ai
    LEFT JOIN groups AS g ON (g.id = ai.group_id AND g.deleted_at IS NULL)
    LEFT JOIN group_users AS gu ON (gu.group_id = g.id)
```

Reply:
agreed, done, thanks!